apps/connect/src/hooks/useConnectCrypto.ts

```diff
@@ -59,11 +58,8 @@ export function useConnectCrypto(): IConnectCrypto {
     );
 }
 
-const didAtom = atom<DID | undefined>(undefined);
-didAtom.debugLabel = 'didAtomInConnect';
-
 export function useConnectDid(): DID | undefined {
```

Comment:
this works as is without an atom. it is a bit wasteful since it is done on every use of the hook, but really it is just a string so it makes no real difference in performance. this can be updated to use my `useSyncExternalStore` solution if needed